src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/LocationsService.kt

```diff
@@ -21,4 +27,24 @@ class LocationsService(
       ?.let { locationsInsidePrisonClient.getVideoLinkLocationsAtPrison(prisonCode) }
       ?.filter { !enabledOnly || it.active }
       ?.toModel() ?: emptyList()
+
+  @Transactional(readOnly = true)
+  fun getDecoratedVideoLocations(prisonCode: String, enabledOnly: Boolean): List<Location> {
+    val prisonLocations = prisonRepository.findByCode(prisonCode)
```

Comment:
Just noticed these lines look the same as that used in the function above `getVideoLinkLocationsAtPrison` could call that here instead of repeating?  meh

Reply:
Yup. Re-used the original to save a line or two.